pkgs/applications/misc/pdfstudio/common.nix

```diff
@@ -0,0 +1,98 @@
+{ pname
+, src
+, year
+, version
+, desktopName
+, pkgs
```

Comment:
It's good practice not to bring `pkgs` into scope, but instead the packages you actually need.
```suggestion
, buildFHSUserEnv
```